From 6db470324ab3dfe66497a1a5a82d1c2a91bbbfd3 Mon Sep 17 00:00:00 2001
From: Everett Hildenbrandt <hildenb2@illinois.edu>
Date: Sat, 9 Sep 2017 21:04:41 -0500
Subject: [PATCH] .gitignore (new): ignoring build products

---
 .gitignore | 34 ++++++++++++++++++++++++++++++++++
 1 file changed, 34 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 00000000..46b5e7e5
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,34 @@
+# metadata files
+*~
+*.sw*
+.DS_Store
+
+# build products (when using automake)
+/autom4te.cache
+/src/Main/maude
+/test-driver
+/aclocal.m4
+/configure
+/config.*
+/.build
+stamp-*
+.dirstamp
+Makefile
+Makefile.in
+.deps
+lib*.a
+
+# testing files
+*.log
+*.out
+*.trs
+
+# generated parser files
+/src/Mixfix/lexer.cc
+/src/Mixfix/surface.h
+/src/Mixfix/surface.output
+/src/Mixfix/surface.yy
+
+# some of the versions of Maude do not correctly sync src/Mixfix/surface.* with the files it's built from
+# until this is fixed, these files will not be ignored
+# src/Mixfix/surface.*
-- 
2.13.5

